src/helpers/storage.js

```diff
@@ -1,34 +1,135 @@
 // eslint-disable-next-line import/prefer-default-export
-import { STORAGE_KEY } from '../config/consts';
+import { STORAGE_TASK_KEY, STORAGE_GROUP_OF_TASKS_KEY } from '../config/consts';
 
 const saveDataInStorageForKey = (key, data) => {
   return localStorage.setItem(key, JSON.stringify(data));
 };
 
 export const getTasksDataFromLocalStorage = () => {
-  return JSON.parse(localStorage.getItem(STORAGE_KEY)) || [];
+  return JSON.parse(localStorage.getItem(STORAGE_TASK_KEY)) || [];
 };
 
 export const addTaskToStorage = (taskTitle) => {
   const currentTasks = getTasksDataFromLocalStorage();
   const newTasks = [...currentTasks, taskTitle];
-  saveDataInStorageForKey(STORAGE_KEY, newTasks);
+  saveDataInStorageForKey(STORAGE_TASK_KEY, newTasks);
 };
 
 export const deleteTaskStorage = (id) => {
   const currentTasks = getTasksDataFromLocalStorage();
   const indexOfDeletedTask = currentTasks.findIndex((task) => task.id === id);
   const newTaskList = currentTasks.filter((task) => task !== currentTasks[indexOfDeletedTask]);
-  saveDataInStorageForKey(STORAGE_KEY, newTaskList);
+  saveDataInStorageForKey(STORAGE_TASK_KEY, newTaskList);
 };
 
 export const updateTaskInStorage = (task) => {
   const currentTasks = getTasksDataFromLocalStorage();
-  const updatedTasks = currentTasks.map((currentTask) => {
-    if (currentTask.id === task.id) {
-      return task;
+  const updatedTasks = currentTasks.map((currentTask) =>
+    currentTask.id === task.id ? task : currentTask,
+  );
+  saveDataInStorageForKey(STORAGE_TASK_KEY, updatedTasks);
+};
+
+export const getGroupOfTasksDataFromLocalStorage = () => {
+  return JSON.parse(localStorage.getItem(STORAGE_GROUP_OF_TASKS_KEY)) || [];
+};
+
+// THIS IS ANOTHER SOLUTION OF updateTaskInGroupStorage LEFT BY PURPOSE
```

Comment:
In code base we do not leave that kind of comments - please remove, before that you can create another branch which will not be merged, but you could still switch and get back to this if you will need

Reply:
done